packages/jetpack-redirects/src/index.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Builds an URL using the jetpack.com/redirect service
+ *
+ * If $source is a simple slug, it will be sent using the source query parameter. e.g. jetpack.com/redirect?source=slug
```

Comment:
Another place to remove the PHP-ish `$source` variable names 🙂

Reply:
fixed in https://github.com/Automattic/wp-calypso/pull/41497/commits/c733a134a39f478a2aa993b814acf9d464ca8719